### PR TITLE
Blog post: Introducing the New JSX Transform

### PR DESCRIPTION
**Pull request by lunaruan**
Blog post for JSX transform

### CODE REVIEW

File: content/blog/2020-09-17-support-for-new-jsx-transform.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+* Because JSX compiled into `React.createElement`, the React library always needed to be in scope if you use JSX. 
+* There are some [performance improvements and simplifications](https://github.com/reactjs/rfcs/blob/createlement-rfc/text/0000-create-element-changes.md#motivation) that `React.createElement` does not allow.
+
+To solve these issues, React 17 introduces two new entry points to the React package that are intended to only be used by compilers like Babel and TypeScript. Instead of transforming JSX to `React.createElement`, the new transform imports special functions from those entry points. These functions will automatically be imported so **React no longer needs to be in scope** to use JSX.
+
+For example,
+
+```js
+function App() {
+  return <div>Hello World</h1>;
```
Comment (jhonmike) [inline content/blog/2020-09-17-support-for-new-jsx-transform.md:55]:
```suggestion
  return <h1>Hello World</h1>;
```

Comment (lunaruan) [reply content/blog/2020-09-17-support-for-new-jsx-transform.md:55]: Oooh good catch. Thanks!

**Review by rickhanlonii — APPROVED** [review]
Looks great!

File: content/blog/2020-09-17-support-for-new-jsx-transform.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
+---
+title: "React v17.0 Release Candidate: Support for New JSX Transform"
```
Comment (gaearon) [inline content/blog/2020-09-17-support-for-new-jsx-transform.md:2]: Maybe "Introducing the New JSX Transform"? I don't think this is very related to 17 RC and might be confusing to see two posts that say they're about 17 RC.

File: content/blog/2020-09-17-support-for-new-jsx-transform.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
+---
```
Comment (gaearon) [inline content/blog/2020-09-17-support-for-new-jsx-transform.md:1]: Meta: let's change the date of the post to Monday.

File: content/blog/2020-09-21-introducing-the-new-jsx-transform.md [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+```json
+"rules": {
+     "react/jsx-uses-react": "off",
+     "react/react-in-jsx-scope": "off"
+}
+```
+
+### TypeScript
+
+TypeScript support for the JSX transform is currently in progress and will be available soon! You can track the progress [here](https://github.com/microsoft/TypeScript/issues/34547).
```
Comment (eps1lon) [inline content/blog/2020-09-21-introducing-the-new-jsx-transform.md:179]: It's part of the 4.1 beta now: https://devblogs.microsoft.com/typescript/announcing-typescript-4-1-beta/#jsx-factories

Comment (gaearon) [reply content/blog/2020-09-21-introducing-the-new-jsx-transform.md:179]: Oh nice. Want to amend the post?

**Review by crypt096 — APPROVED** [review]

**Pull request merged**